dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/OSUtils.java

```diff
@@ -134,7 +135,11 @@ public static double loadAverage() {
      */
     public static double cpuUsage() {
         CentralProcessor processor = hal.getProcessor();
-        double cpuUsage = processor.getSystemCpuLoad();
+
+        long[] preTicks = processor.getSystemCpuLoadTicks();
+        Util.sleep(1000);
```

Comment:
Why we should sleep here?

Reply:
According to the method annotation, getSystemCpuLoadTicks() method  returns the "recent cpu usage" for the whole system by counting ticks between the user-provided value from a previous call. If not sleeping, it will always return value with 0.0D
`
    /**
     * Returns the "recent cpu usage" for the whole system by counting ticks from
     * {@link #getSystemCpuLoadTicks()} between the user-provided value from a
     * previous call.
     *
     * @param oldTicks
     *            A tick array from a previous call to
     *            {@link #getSystemCpuLoadTicks()}
     * @return CPU load between 0 and 1 (100%)
     */
    double getSystemCpuLoadBetweenTicks(long[] oldTicks);`